### DISCUSSION TITLE
Sort result of API call

### DISCUSSION THREAD

**Discussion by styxiner** [wazuh/wazuh#22851] [Q&A]
I'm trying to retrive the fields cve, severity and name of an agent on that exact order. I put this on the API console:
`GET /vulnerability/003?q=severity=Critical&select=cve,severity,name&sort=+cve,severity,name`
but it returns it with the oposite order: name, severity and cve. Can I do something with the API call or should I just give it a format on the output?
![image](https://github.com/wazuh/wazuh/assets/160549592/958a712f-0937-4895-94dd-01c646c3a91a)

**Comment by MiguelazoDS** [comment]
Hi @Jaime-Moranchel 

Sorry for the delay, you probably found the answer by now, but the sorting feature you mentioned sorts the keys according to their value, it does not sort by key name. Why is that required? The good about JSON is that you get the value specified by the key, you do not need to have that in a specific order. Beyond that, I'd try with JQ tool.

**Comment by sebasfalcone** [comment]
Inactive thread, please re-open if necessary
